ui/apps/platform/src/Containers/Clusters/AddClusterPrompt.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import {
+    Alert,
+    AlertVariant,
+    Button,
+    ButtonVariant,
+    Flex,
+    FlexItem,
+    Text,
+    TextContent,
+    TextVariants,
+} from '@patternfly/react-core';
+
+import SecurityFullColorPink from 'images/Security_FullColor_Pink.svg';
+
+function AddClusterPrompt() {
+    return (
+        <>
+            <Alert isInline variant={AlertVariant.success} title="You are ready to go!">
+                <p className="pf-u-font-weight-normal">
+                    You have successfully deployed a Red Hat Advanced Cluster Security platform. Now
+                    you can configure the clusters you want to secure.
+                </p>
+            </Alert>
+            <Flex
+                alignItems={{ default: 'alignItemsCenter' }}
+                justifyContent={{ default: 'justifyContentCenter' }}
+                className="pf-u-text-align-center"
+                direction={{ default: 'column' }}
```

Comment:
Is this equivalent to `Bullseye` layout element?

https://www.patternfly.org/v4/layouts/bullseye/design-guidelines

> Use a **Bullseye** layout to center content, both vertically and horizontally within a container.

Reply:
I tried a Bullseye at first, but required style overrides to get the spacing between the image, text, and button correct. Rather than hack in overrides, I chose to simplify. (The larger UX team recommended this look to Zhenpeng as a variation the